Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/messaging/service/decorator/HttpHeaderServiceEndpointDecoratorTest.java

```diff
@@ -101,6 +101,40 @@ protected PropertyAccessor getPropertyAccessor() {
     }
     
     @Test
+    public void testConfigure_WithoutValues() throws PropertyAccessException {
+        CONNECTTestClient<TestServicePortType> testClient = new CONNECTTestClient<>(
+                new TestServicePortDescriptor());
+
+        ServiceEndpoint<TestServicePortType> serviceEndpoint = testClient.getServiceEndpoint();
+        
+        final PropertyAccessor mockPropAccessor = mock(PropertyAccessor.class);
+
+        AssertionType assertion = new AssertionType();
+        assertion.setKeepAlive("true");
+        
+        CONNECTCustomHttpHeadersType assertionHeaders = new CONNECTCustomHttpHeadersType();
+        assertionHeaders.setHeaderName("headerName");
+        assertionHeaders.setHeaderValue("headerValue");
+        
+        assertion.getCONNECTCustomHttpHeaders().add(assertionHeaders);
+
+       HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(serviceEndpoint, assertion) {
+            
+  /*         @Override
+   *         protected PropertyAccessor getPropertyAccessor() {
+   *            return mockPropAccessor;
+   *         }
+      
+ */     };
+        
+   /*     when(mockPropAccessor.getPropertyNames(NhincConstants.GATEWAY_PROPERTY_FILE)).thenReturn(new HashSet<String>());
+        */       
+        headerDecorator.configure();
+        
+        validateConfiguration(headerDecorator); 
```

Comment:
You can add an  assert statement to make this method test the actual functional logic assertEquals(headerDecorator.getHTTPClientPolicy().getConnection(),ConnectionType.KEEP_ALIVE)